awx/ui_next/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.jsx

```diff
@@ -1,23 +1,33 @@
-import React, { useEffect, useRef, useState } from 'react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
 import { Link, useHistory } from 'react-router-dom';
 import { withI18n } from '@lingui/react';
 import { t } from '@lingui/macro';
+import styled from 'styled-components';
+import { QuestionCircleIcon as PFQuestionCircleIcon } from '@patternfly/react-icons';
 
-import { Button, Chip, List, ListItem } from '@patternfly/react-core';
+import { Button, Chip, List, ListItem, Tooltip } from '@patternfly/react-core';
 import AlertModal from '../../../components/AlertModal';
 import { CardBody, CardActionsRow } from '../../../components/Card';
 import ChipGroup from '../../../components/ChipGroup';
 import { VariablesDetail } from '../../../components/CodeMirrorInput';
+import ContentError from '../../../components/ContentError';
+import ContentLoading from '../../../components/ContentLoading';
 import CredentialChip from '../../../components/CredentialChip';
 import DeleteButton from '../../../components/DeleteButton';
+import InventorySourceSyncButton from '../shared/InventorySourceSyncButton';
 import {
   DetailList,
   Detail,
   UserDateDetail,
 } from '../../../components/DetailList';
 import ErrorDetail from '../../../components/ErrorDetail';
+import useRequest from '../../../util/useRequest';
 import { InventorySourcesAPI } from '../../../api';
 
+const QuestionCircleIcon = styled(PFQuestionCircleIcon)`
+  margin-left: 10px;
+`;
+
```

Comment:
We have a `<FieldTooltip>` component that takes care of this and some mouseover behavior we want to keep consistent between tooltips. (components/FormField/FieldTooltip)

Reply:
good catch